src/const.js

```diff
@@ -5,24 +5,22 @@
 
 var CONST = {};
 
-CONST.DOM_SANDBOX_OVERRIDE_DOM_METHOD_NAME = 'tc_odm_e01a2ff5';
-CONST.DOM_SANDBOX_PROCESSED_CONTEXT        = 'tc-spc-c1208da2';
-CONST.DOM_SANDBOX_STORED_ATTR_POSTFIX      = '_e01a2f_stored';
-CONST.EVENT_SANDBOX_WHICH_PROPERTY_WRAPPER = 'tc-e01a2f-which';
+CONST.DOM_SANDBOX_OVERRIDE_DOM_METHOD_NAME = '@hammerhead-override-dom@';
+CONST.DOM_SANDBOX_PROCESSED_CONTEXT        = '@hammerhead|processed-context@';
+CONST.EVENT_SANDBOX_WHICH_PROPERTY_WRAPPER = '@hammerhead|which-property-wrapper@';
+CONST.DOM_SANDBOX_STORED_ATTR_POSTFIX      = '-hammerhead-stored-value';
 
 CONST.IS_STYLESHEET_PROCESSED_COMMENT = '/* stylesheet processed via hammerhead */';
 
-CONST.UPLOAD_SANDBOX_HIDDEN_INPUT_NAME = 'upload-89fc3-info';
+CONST.UPLOAD_SANDBOX_HIDDEN_INPUT_NAME = '@hammerhead|upload-info-hidden-input@';
 
-CONST.HOVER_PSEUDO_CLASS_ATTR = 'data-1b082a6cec-hover';
+CONST.HOVER_PSEUDO_CLASS_ATTR = 'data-hammerhead-hovered';
 
 CONST.XHR_CORS_SUPPORTED_FLAG   = 0x10;
-CONST.XHR_REQUEST_MARKER_HEADER = 'x-tc-xm-cd46977f';
+CONST.XHR_REQUEST_MARKER_HEADER = '%hammerhead|xhr-request-marker-header%';
```

Comment:
You mean `@`? It's used in HTTP header name and not URL AFAIK and I see that `@` is disallowed there: http://www.w3.org/Protocols/rfc2616/rfc2616-sec2.html#sec2.2
We can get rid of prefix and postfixes then and just keep name it `hammerhead|xhr-request-marker-header`. What do you think?